notes/gcc-14.md

```diff
@@ -19,7 +19,7 @@ GNU gcc warning that now promote to errors in any mode higher than C90:
 
 ### -Wimplicit-function-declaration
 
-Functions can no longer be called without be declaring first.  Fixing this may
+Functions can no longer be called without be declared first.  Fixing this may
```

Comment:
s/without be declared/without being declared/

Reply:
This has already been integrated in c4788b1037574820a74127eb928bf176f3ed4477

Reply:
done